integration_tests/assets/amid_data/mock-wazo-amid.py

```diff
@@ -38,9 +43,9 @@ def log_request():
 
     log = {'method': request.method,
            'path': request.path,
-           'query': request.args.items(multi=True),
-           'body': request.data,
-           'json': request.json,
+           'query': dict(request.args.items(multi=True)),
```

Comment:
Yes, that is the case. I only converted to dict because the previous object was not serializable. Yes, if we need all values we can instead use the `dict(request.args.lists())` however, that would have changed the format as it would no longer be key/value pairs. It is rare to send keys multiple times so this is the most common way. In these tests this value isn't even used, it is merely stored.
Don't forget these are tests in a controlled environment.